README.md

```diff
@@ -1,223 +1,186 @@
-for example UART, sockets or for example stdio (process `DUT`).
-
-A more detailed description of the *Icetea* concept is
-available [here](doc/README.md).
-
-## Installation
-
-`> python setup.py install`
-
-When installing Icetea, installing to a
-[virtualenv](https://virtualenv.pypa.io/en/stable/installation/) is
-a good idea.
-
-### Required dependencies:
-
-* In case you want to use **local mbed devices**
-  * [mbed-ls](https://github.com/armmbed/mbed-ls)
-    * To enumerate connected boards
-  * [mbed-flasher](https://github.com/ARMmbed/mbed-flasher)
-    * to flash devices automatically
-  * Both of these should be installed automatically by
-  Icetea installation.
-
-**OS specific:**
-
-Linux:
-* python-dev and python-lxml
-`sudo apt-get install python-dev python-lxml`
-* For documentation install sphinx:
-`apt-get install python-sphinx`
-
-OS X:
-* [XCode developer tools](http://osxdaily.com/2014/02/12/install-command-line-tools-mac-os-x/)
-* [MacPorts](https://www.macports.org/install.php)
-* lxml as described
-[here](http://lxml.de/installation.html#installation):
-`STATIC_DEPS=true sudo pip install lxml`
-* For documentation install sphinx:
-`sudo port install py27-sphinx`, `sudo port select --set python python27`,
-`sudo port select --set sphinx py27-sphinx`
-
-Windows:
-* python-lxml installation is problematic on Windows since
-it usually requires build tools. It can however be installed
-from pre-built binaries.
-    * Download binary for you system from the internet.
-    * Navigate to the directory where you downloaded the
-    binary and install it with `pip install <insert_file_name>`
-* For documentation install sphinx with pip:
-`pip install sphinx`
-
-### Optional dependencies
+the command line interface (`CLI`)
+in a device under test (`DUT`). The interface between the test framework and `DUT` can be
+for example UART or for example stdio (process `DUT`).
+
+More detailed documentation and information on deeper details of
+the framework is available
+[here in rst format](https://github.com/ARMmbed/icetea/tree/master/doc-source).
+and [here in markdown format](https://github.com/ARMmbed/icetea/tree/master/doc).
+
+### Installation
+
+`> pip install icetea`
+
+#### Prerequisites
+Icetea supports Linux (Ubuntu preferred), Windows and OS X. Our main target is Linux.
+We support both python 2.7 and 3. Some OS specific prerequisites below:
+
+* Linux
+    * python-dev and python-lxml
+        `sudo apt-get install python-dev python-lxml`
+    * In order to run test cases with hardware in Linux without sudo rights:
+        ```
+        sudo usermod -a -G dialout username
+        Log out & log in back to Linux
+        ```
+        This command will add the user 'username' to the 'dialout' group and
+        grant the required permissions to the USB ports.
+* OS X
+    * [XCode developer tools](http://osxdaily.com/2014/02/12/install-command-line-tools-mac-os-x/)
+    * [MacPorts](https://www.macports.org/install.php)
```

Comment:
Why we need macPorts? I don't see any reason

Reply:
true - This is not needed - at least I haven't need it so far. Can be removed.

Reply:
I rely entirely on others to tell me how the mac installation works since I'm not a mac user myself. @jupe @RomanSaveljev either of you remember why MacPorts was added to OS X requirements?